### PR TITLE
Fix warnings and more

### PR DESCRIPTION
**Pull request by etchegom**
- fixing some warnings appearing in test logs 
- preparing to django 5 upgrade by suppressing deprecation warnings

### CODE REVIEW

File: conventions/services/search.py [hunk 1]
```diff
@@ -50,6 +50,8 @@ def get_queryset(self) -> QuerySet:
 
         if order_by := self._get_order_by():
             queryset = queryset.order_by(*order_by)
+        else:
+            queryset = queryset.order_by("-cree_le")
```
Comment (etchegom) [inline conventions/services/search.py:54]: We need an ordered queryset any time.
Actually, it is always specified explicitely, except for `Programme`, but we don't really care about the order for this case, as we return only a few items.

File: conventions/templatetags/custom_filters.py [hunk 2]
```diff
@@ -500,3 +500,11 @@ def get_ordered_full_path(request, order_field):
...
     get_copy["page"] = 1
     return f"{request.path}?{get_copy.urlencode()}"
+
+
+@register.filter
+def length_is(value, arg):
+    try:
+        return len(value) == int(arg)
+    except (ValueError, TypeError):
+        return False
```
Comment (etchegom) [inline conventions/templatetags/custom_filters.py:510]: https://stackoverflow.com/questions/78874958/invalid-filter-length-is-error-in-django-template-how-to-fix

File: conventions/services/search.py (hunk 1 above)
Comment (kolok) [reply conventions/services/search.py:54]: 👍

**Review by kolok — APPROVED** [review]
🌷

**Pull request merged**